appservice/src/createAppService/createAppService.ts

```diff
@@ -92,9 +89,15 @@ export async function createAppService(
         default:
     }
     executeSteps.push(new SiteCreateStep(createOptions.createFunctionAppSettings));
-    const wizard: AzureWizard<IAppServiceWizardContext> = new AzureWizard(promptSteps, executeSteps, wizardContext);
 
-    wizardContext = await wizard.prompt(actionContext);
+    if (wizardContext.newSiteOS !== undefined) {
+        SiteOSStep.setLocationsTask(wizardContext);
+    }
+
+    const title: string = localize('creatingTitle', 'Create new {0}', getAppKindDisplayName(appKind));
```

Comment:
This can cause trouble with localization.  E.g. in German the spelling of "new" here will depend on the specific app kind's gender.  It would be better to pass in the entire creatingTitle as a phrase rather than a piece of a sentence.